state/stateSaver.go

```diff
@@ -138,10 +138,8 @@ func SaveToFile(s *State, dbht uint32, b []byte, filename string) error {
 }
 
 func LoadFromFile(s *State, filename string) ([]byte, error) {
-	fmt.Fprintf(os.Stderr, "%20s Load state from %s\n", s.FactomNodeName, filename)
 	b, err := ioutil.ReadFile(filename)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "%20s LoadFromFile error: %v\n", s.FactomNodeName, err)
```

Comment:
those 2 lines are already been logged by the calling function (just with a different wording)